api/zsock.api

```diff
@@ -126,6 +126,16 @@
         <argument name = "endpoint" type = "string" />
     </constructor>
 
+    <constructor name = "new gather" state = "draft">
+        Create a GATHER socket. Default action is bind.
+        <argument name = "endpoint" type = "string" />
+    </constructor>
+
+    <constructor name = "new scatter" state = "draft">
+        Create a GATHER socket. Default action is connect.
```

Comment:
@sappo - hah, thanks! I don't get to push that green button often enough, so have itchy trigger finger when it comes to Pulls. :)